testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/resource/patch/StudentJsonMergePatchTest.java

```diff
@@ -0,0 +1,198 @@
+package org.jboss.resteasy.test.resource.patch;
+
+import jakarta.json.Json;
+import jakarta.json.JsonObject;
+import jakarta.ws.rs.HttpMethod;
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.client.Entity;
+import jakarta.ws.rs.client.WebTarget;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.jboss.arquillian.container.test.api.Deployer;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.OperateOnDeployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit5.ArquillianExtension;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.utils.PortProviderUtil;
+import org.jboss.resteasy.utils.TestUtil;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Tests related to JSON Merge Patch
+ * @see <a href="https://tools.ietf.org/html/rfc7386">JSON Merge Patch</a>
+ * @see <a href="https://issues.redhat.com/browse/RESTEASY-2567">RESTEASY-2567</a>
+ */
+@ExtendWith(ArquillianExtension.class)
+@RunAsClient
+public class StudentJsonMergePatchTest {
+
+    private static final Logger log = Logger.getLogger(StudentJsonMergePatchTest.class);
+
+    @ArquillianResource
+    private Deployer deployer;
+
+    static Client client;
+    static final String PATCH_DEPLOYMENT = "Patch";
+    static final Student testStudentEntity = new Student()
```

Comment:
Minor nit, there should be no reason for these not to be `private`.

Reply:
Thanks, fixed.